ext/cl_khr_command_buffer_mutable_dispatch.asciidoc

```diff
@@ -233,6 +236,7 @@ CL_DEVICE_MUTABLE_DISPATCH_CAPABILITIES_KHR        0x12B0
 
 // Property to cl_ndrange_kernel_command_properties_khr
 CL_MUTABLE_DISPATCH_UPDATABLE_FIELDS_KHR           0x12B1
+CL_MUTABLE_DISPATCH_PROMISES_KHR                   0x12B2
```

Comment:
The ``0x12B2` value is already taken https://github.com/KhronosGroup/OpenCL-Docs/blob/main/xml/cl.xml#L1777 , `0x12B7` looks like the next available value to use